httpclientasync-scenario/src/main/java/org/apache/skywalking/testcase/httpclientasync/BackController.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.skywalking.testcase.httpclientasync;
+
+import java.sql.SQLException;
+import org.apache.log4j.Logger;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@EnableAutoConfiguration()

```

Comment:
Is `@EnableAutoConfiguration` necessary? if not, remove it.

Reply:
'@EnableAutoConfiguration()'  is necessary.

Reply:
Why don't you move the `@EnableAutoConfiguration` to `HttpasyncclientApplication` class, You don't necessary to write this twice.